openapi.yaml

```diff
@@ -41,17 +45,9 @@ components:
 
     LinkedContentItem:
       required:
-        - analytics_identifier
-        - api_path
-        - base_path
         - content_id
-        - description
-        - document_type
         - locale
-        - public_updated_at
-        - schema_name
         - title
```

Comment:
I've been trying to work out what is actually required here - and it's turned up to be a bit confusing (unsurprisingly). 

Looks like I was wrong about these being not required and instead may come out as nil:

```
[{:analytics_identifier=>nil,
  :api_path=>nil,
  :base_path=>nil,
  :content_id=>"c6e33b13-42a8-4431-980d-40bd55ec4518",
  :description=>nil,
  :document_type=>"contact",
  :locale=>"en",
  :public_updated_at=>"2016-05-23T14:22:13Z",
  :schema_name=>"contact",
  :title=>"Inward Investment Enquiries",
  :withdrawn=>false,
  :links=>{}}]
```